google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1654,6 +1699,21 @@ private ApiTracerFactory getDefaultApiTracerFactory() {
     return BaseApiTracerFactory.getInstance();
   }
 
+  private ApiTracerFactory getMetricsApiTracerFactory(boolean isDirectPathChannelCreated) {
```

Comment:
Rename to `createMetricsApiTracerFactory` to make clear that it creates a new instance every time it is called.